src/cmd/cli/command/compose.go

```diff
@@ -79,8 +79,8 @@ func makeComposeUpCmd() *cobra.Command {
 
 			numGPUS := compose.GetNumOfGPUs(cmd.Context(), project)
 			if numGPUS > 0 {
-				if err := gating.HasAuthorization(gating.ResourceGPU, "usage of GPUs. To resolve see https://docs.defang.io/docs/tutorials/deploy-with-gpu"); err != nil {
-					return err
+				if !gating.Gates.Gpu {
+					return gating.ErrNoPermission("usage of GPUs. To resolve see https://docs.defang.io/docs/tutorials/deploy-with-gpu")
```

Comment:
wrong link? This tells how to increase spot quotas, but the error is a tier thing

Reply:
I also suggested linking to `https://portal.defang.dev/pricing`, and as soon as I did, I kind of cringed that the url was `/pricing`. Do you think the url should maybe be something less 🤑🤑🤑  like `/subscription`? @lionello

Reply:
Agree. @raphaeltm ?